ext/storage_sources/azure_store/test/test_azure_connection.cpp

```diff
@@ -73,9 +73,10 @@ TEST_CASE("Testing Azure Connection Class", "azure-connection")
       std::getenv("AZURE_STORAGE_CONNECTION_STRING"), "myblobcontainer1");
     bool exists = false;
 
-    std::string obj_prefix = randomize_test_prefix();
+    std::string bucket_prefix = randomize_test_prefix();
 
-    azure_connection conn = azure_connection("myblobcontainer1", obj_prefix);
+    azure_connection conn = azure_connection("myblobcontainer1", bucket_prefix);
+    azure_connection conn_second = azure_connection("myblobcontainer1", "pfx_");
```

Comment:
I don't think we need the second connection class here.

Reply:
yup removed :)